runtime/vm/OutOfLineINL_openj9_internal_foreign_abi_InternalUpcallHandler.cpp

```diff
@@ -32,7 +32,7 @@ extern "C" {
 
 #if JAVA_SPEC_VERSION >= 16
 /**
- * openj9.internal.foreign.abi.InternalUpcallHandler: private native long allocateUpcallStub(UpcallMHMetaData mhMetaData, String[] cSignatureStrs);
+ * openj9.internal.foreign.abi.InternalUpcallHandler: private native synchronized long allocateUpcallStub(UpcallMHMetaData mhMetaData, String[] cSignatureStrs);
```

Comment:
The cast on line 141 should be `(I_64)(intptr_t)thunkAddr`, not `(I_64)(uintptr_t)thunkAddr`.

Reply:
Fixed as suggested above.